src/components/hoc/styled.tsx

```diff
@@ -1,26 +1,72 @@
 import * as React from 'react'
-import { ImageStyle, StyleProp, TextStyle, ViewStyle } from 'react-native'
+import { ImageStyle, StyleProp, StyleSheet, TextStyle, ViewStyle } from 'react-native'
 
-import { Theme, useTheme } from '../services/ThemeContext'
+import { cacheStyles, getTheme, Theme, useTheme } from '../services/ThemeContext'
 
 interface StyleProps {
   style?: StyleProp<any>
 }
-type ValidStyles = ImageStyle | TextStyle | ViewStyle | ValidStyles[]
 
+export type ValidStyles = ImageStyle | TextStyle | ViewStyle
```

Comment:
Optional: We put in this export while we were debugging, but no external files use it anymore. This type could be private again.